modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -9,11 +9,19 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* If your goal is a faster installation, use step 1 and create your own minimal blueprint file.
+* If your goal is a faster startup for {microshift-short}, use the generated blueprint in step 2. Using the generated blueprints is a longer installation process, but a faster start up because container references are accessed locally.
```

Comment:
Faster installation and faster startup are by-products of having one or another blueprint type, but they are not goals.
I think we should rephrase this to emphasize that step 1 results a minimal microshift installation, while the step 2  includes all the RPM packages and container images. We can still mention that the former results in a faster installation, and the latter in faster startup.